src/rom_8011DC0.c

```diff
@@ -182,17 +180,18 @@ struct TradeUnkStruct
 };
 
 extern struct TradeUnkStruct gUnknown_02022C40;
-
+extern struct UnkLinkRfuStruct_02022B14Substruct gUnknown_02022C38;
 extern union UnkUnion_Main gUnknown_02022C30;
-
 extern u8 gUnknown_02022C2C;
 extern u8 gUnknown_02022C2D;
 extern u8 gUnknown_02022C3E;
 extern u16 gUnknown_02022C3C;
-extern struct UnkLinkRfuStruct_02022B14Substruct gUnknown_02022C38;
-
+extern u8 gUnknown_02022C20[];
 extern u8 gFieldLinkPlayerCount;
 extern u8 gUnknown_03005DB4;
+extern u8 gSelectedOrderFromParty[];
+extern struct MailStruct gUnknown_020321C0[PARTY_SIZE];
+extern u8 gUnknown_02032298[2];
 
 // IWRAM vars
 IWRAM_DATA struct UnkStruct_Leader *gUnknown_03000DA0;
```

Comment:
Please use `BSS_DATA` for forwards compatibility.

Reply:
IMO we should left this once we've got everything in src.